src/test/components/TrackThread.test.js

```diff
@@ -188,6 +191,19 @@ describe('timeline/TrackThread', function () {
     expect(getCallNodePath()).toEqual(['j', 'k', 'l']);
   });
 
+  it('when clicking a stack, this selects the call tree panel', function () {
```

Comment:
I couldn't find a test using the activity graph, but using the stack graph canvas works too :-D